project/plugins.sbt

```diff
@@ -1,5 +1,3 @@
-libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
-
-addSbtPlugin("org.typelevel"  % "sbt-typelevel-sonatype-ci-release" % "0.5.0-RC5")
```

Comment:
We shouldn't be using sbt-typelevel-sonatype-ci-release since this is specific to typelevel projects, instead we should use sbt-github-actions directly which has all of the same functionality (and is actually maintained).